packages/types/src/srml/parachains/types.ts

```diff
@@ -0,0 +1,167 @@
+// Auto-generated, do not edit
+
+import { Enum, Struct, Tuple, Vector } from '../../codec';
```

Comment:
Only allowed imports are from codec and primitives. (There needs to be a way to link across modules as well, not available as of yet, i.e. `AuthorityId` would be imported from e.g. `grandpa`)